src/app/shared/components/basket/basket-cost-center-selection/basket-cost-center-selection.component.ts

```diff
@@ -71,11 +55,25 @@ export class BasketCostCenterSelectionComponent implements OnInit, OnDestroy {
         whenTruthy(),
         map(val => val.costCenter),
         distinctUntilChanged(),
+        withLatestFrom(this.checkoutFacade.basket$),
         takeUntil(this.destroy$)
       )
-      .subscribe(costCenter => {
-        this.checkoutFacade.updateBasketCostCenter(costCenter);
+      .subscribe(([costCenter, basket]) => {
+        if (costCenter !== basket.costCenter) {
```

Comment:
This condition has to be a bit more precise - "", undefined and null values have to be treated as the same, if a user has more than one cost center 2 requests are send if the user enters cart page setting the cost center null and "" , but actually no request should be sent in this case, see image